packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -112,7 +113,8 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
               aria-haspopup='true'
               autoCapitalize='off'
               autoComplete='off'
-              role='combobox' />
+              role='combobox'
+              disabled={ this.state.items.length >= itemLimit } />
```

Comment:
For 1. I can add a maxItemLimit property or something like that.

For 2. You are definitely correct, that is a bug.